Realm/Realm/InteropConfig.cs

```diff
@@ -70,6 +70,18 @@ public static string DefaultStorageFolder
             set => _customStorageFolder = value;
         }
 
+        public static string TryDefaultStorageFolder(string type)
+        {
+            try
+            {
+                return DefaultStorageFolder;
+            }
+            catch (InvalidOperationException e)
```

Comment:
```suggestion
            catch (InvalidOperationException)
```
Since you're not using the exception itself